airbyte-lib/README.md

```diff
@@ -9,6 +9,17 @@ airbyte-lib is a library that allows to run Airbyte syncs embedded into any Pyth
 * For examples, check out the `examples` folder. They can be run via `poetry run python examples/<example file>`
 * Unit tests and type checks can be run via `poetry run pytest`
 
+## Release
+
+* In your PR:
+   * Bump the version in `pyproject.toml`
+   * Add a changelog entry to the table below
+* Once the PR is merged, go to Github and trigger the `Publish AirbyteLib Manually` workflow. This will publish the new version to PyPI.
```

Comment:
Do we get CI checks for missing changelog entries and/or unbumped versions?


It would be great if we could piggy-back on existing rails, if that's possible.

Reply:
There's not really good existing guardrails as all places kind of do their own thing (connectors, airbyte-cdk). However, I added a basic check that a changelog entry is set for the current version.

I kept this simple deliberately so we have control over the publishing process in the first messy phase. We can add some automation later on if it makes sense (like auto-publishing as soon as the version changes on merge to master).